imi/src/main/java/org/motechproject/nms/imi/service/impl/CdrFileServiceImpl.java

```diff
@@ -141,14 +229,21 @@ private ParseResults parseDetailRecords(File file, FileInfo fileInfo) {
 
             String line;
             while ((line = reader.readLine()) != null) {
-
                 try {
                     CallDetailRecordDto cdr = CsvHelper.csvLineToCdr(line);
 
-                    parseDetailRecord(cdr, records);
+                    if (!verifyOnly) {
```

Comment:
nit pick, but reading this as 'if not verifyOnly' confuses me.  I wonder if createing a local boolean at the type like this:

boolean provideAggregation = !verifyOnly

Then this if would read as 'if provideAggregation' which is less confusion (to me).  Either that or change the parameter but I'm not sure how that would read to the caller.  

Again this is a nit pick.